Doc/library/stdtypes.rst

```diff
@@ -4823,7 +4823,13 @@ can be used interchangeably to index the same dictionary entry.
    being added is already present, the value from the keyword argument
    replaces the value from the positional argument.
 
-   To illustrate, the following examples all return a dictionary equal to
+   Providing keyword arguments as in the first example only works for keys that
+   are valid Python identifiers.  Otherwise, any valid keys can be used.
+   
+   Dictionaries compare equal if and only if they have the same ``(key,
```

Comment:
```suggestion
   Providing keyword arguments as in the first example only works for keys that
   are valid Python identifiers.  Otherwise, any valid keys can be used.

   Dictionaries compare equal if and only if they have the same ``(key,
```